packages/expo-updates/ios/EXUpdates/Update/EXUpdatesUpdate.m

```diff
@@ -58,16 +58,26 @@ + (instancetype)updateWithManifest:(NSDictionary *)manifest
                           response:(nullable NSURLResponse *)response
                             config:(EXUpdatesConfig *)config
                           database:(EXUpdatesDatabase *)database
+                             error:(NSError **)error
 {
-  if (config.usesLegacyManifest) {
+  NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *)response;
+  NSDictionary *headerDictionary = [httpResponse allHeaderFields];
+  NSNumber *expoProtocolVersion = headerDictionary[@"expo-protocol-version"];
```

Comment:
should this be an NSString?

Reply:
The intention is to have it as a number server side and the conversion works as written (manual tests). 

Is there a reason related to header formatting or something that I am missing?

Reply:
Can headers have types other than strings? I think this line might just be mislabeling the actual type; the conversion below would work either way because NSString also happens to have a property called `integerValue`.